source/get-started/connect-to-mongodb.txt

```diff
@@ -0,0 +1,83 @@
+.. _java-rs-get-started-connect-to-mongodb:
+
+==================
+Connect to MongoDB
+==================
+
+.. TODO: 
+.. Add this sentence after the last sentence in the paragraph for implementing reactive streams subscribers. 
+.. To learn more about the Java Reactive Streams API, see link here.
+
+The {+driver-short+} is an implementation of the Reactive Streams API, which consists of three components: 
+
+- ``Publisher``
+- ``Subscriber``
+- ``Subscription``
+
+A ``Publisher`` is a provider of an unbounded number of sequenced elements, published
+according to the demand received from its ``Subscriber`` or multiple instances
+of ``Subscriber``. A ``Subscription`` represents a one-to-one lifecycle of a ``Subscriber`` subscribing to a ``Publisher``.
+
+.. tip::
+
+   To learn more about reactive streams, visit
+   the `Reactive Streams documentation <http://www.reactive-streams.org>`__.
+
+In this tutorial, you must implement {+java-rs+} ``Subscribers`` to
+query your database. You must also use an integrated development environment
+(IDE) to complete the tutorial. This guide uses methods from the Reactor library, a library
+based on the Reactive Streams specification to implement {+java-rs+}
+``Subscribers``. 
```

Comment:
S: Right now, this paragraph is describing a lot of different aspects of the sample app. Can you organize this information into the steps where it is relevant or restructure the paragraph to be more cohesive, perhaps as a list of items to keep in mind or something?